test/e2e/service_latency.go

```diff
@@ -85,45 +90,61 @@ var _ = Describe("Service endpoints latency", func() {
 		Logf("50 %%ile: %v", p50)
 		Logf("90 %%ile: %v", p90)
 		Logf("99 %%ile: %v", p99)
+		Logf("Total sample count: %v", len(dSorted))
 
-		if p99 > 4*p50 {
-			failing.Insert("Tail latency is > 4x median latency")
+		if p50 > time.Second*30 {
```

Comment:
Yeah actually I'm seeing 15s, which is 3-4x what it used to be, and I agree that's too long. But I would like to get this in to keep it from getting worse. I can file an issue so we can track getting that back down again.

I'll change the conditions to be a constant.